sos/report/plugins/ceph.py

```diff
@@ -138,4 +148,32 @@ def setup(self):
             "/etc/ceph/*bindpass*"
         ])
 
+        # If containerized, run commands in containers
+        containers_list = self.list_ceph_containers("ceph-*")
```

Comment:
Why not just call `self.get_all_containers_by_regex("ceph-*")` directly here?

Reply:
Because at first I thought that I might have to use it multiple times and at the end ... I didn't. Thx for the highlight.